Bottom: 8748aed9cab51a726d75b11fdf68618941ea011c
Top:    d921b80822be787fe0691fc32b2d605818b2a00c
Author: Robert Collins <robertc@robertcollins.net>
Date:   2013-03-06 22:47:35 +1300

Refresh of stream-summary

---

diff --git a/testtools/testresult/real.py b/testtools/testresult/real.py
index f89547f..4f8408e 100644
--- a/testtools/testresult/real.py
+++ b/testtools/testresult/real.py
@@ -505,7 +505,7 @@ _status_map = {
 
 
 def test_dict_to_case(test_dict):
-    """Convert a test dict into a TestCase objection.
+    """Convert a test dict into a TestCase object.
 
     :param test_dict: A test dict as generated by StreamToDict.
     :return: A PlaceHolder test object.
